### PR TITLE
Actually fix APM

### PR DESCRIPTION
**Pull request by jamieparkinson**
Turned out server-side APM needs to provide an API key - I've actually tested it this time rather than guessing!

### CODE REVIEW

File: common/services/apm/apmConfig.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
   serverUrl: process.env.APM_SERVER_URL,
   // we've used inactive so by default it is on, and we can purposefully turn it off locally
   active: process.env.APM_INACTIVE !== 'true',
   centralConfig: true,
 });
+
+const client = commonApmConfig;
+const server = serviceName => ({
+  ...commonApmConfig(serviceName),
+  secretToken: process.env.APM_SECRET,
```
Comment (jamesgorrie) [inline common/services/apm/apmConfig.js:13]: Do we not need to add this to the terraform of the apps then to exist? Assuming it just exists on your local machine?

Comment (jamieparkinson) [reply common/services/apm/apmConfig.js:13]: Ha, yes, knew I was missing something

Comment (jamieparkinson) [reply common/services/apm/apmConfig.js:13]: Turned out we'd never added the config for RUM to the identity app TF either!

**Review by jamesgorrie — APPROVED** [review]

File: common/services/apm/apmConfig.js (hunk 1 above)
Comment (jamesgorrie) [reply common/services/apm/apmConfig.js:13]: I really like the way vercel uses a `vercel env` command to generate values locally from remote... something to look into.

https://vercel.com/docs/concepts/projects/environment-variables

**Pull request merged**